code/datums/uplink/exosuit_equipment.dm

```diff
@@ -6,75 +6,75 @@
 /datum/uplink_item/item/exosuit_equipment/mounted_gun
 	name = "Mounted Electrolaser Carbine"
 	desc = "A dual fire mode electrolaser system connected to the exosuit's targetting system."
-	item_cost = 4
+	item_cost = 2
 	path = /obj/item/mecha_equipment/mounted_system/combat/taser
 
 /datum/uplink_item/item/exosuit_equipment/mounted_gun/ion
 	name = "Mounted Ion Rifle"
 	desc = "An exosuit-mounted ion rifle. Handle with care."
-	item_cost = 12
+	item_cost = 3
```

Comment:
```suggestion
	item_cost = 7
```
The ion rifle is extremely capable against the current (default) security armament.
Therefire it should require a significant investment in TCs